### ISSUE TITLE
ValueError: x and y arrays must have at least 2 entries

### ISSUE THREAD

**Issue by zeeshankhan2** [polca/premise#116]
Hi there,
I am trying to make new database from default scenarios but i am getting the following error. It worked out perfectly on another computer but on my PC it is giving following error.

![image](https://github.com/polca/premise/assets/86916007/8625e043-2c7a-4984-9bdd-bdd026253500)
![image](https://github.com/polca/premise/assets/86916007/0c3ed41d-9db0-42a8-9d4a-77208e6d42fc)
![image](https://github.com/polca/premise/assets/86916007/3f9cfa04-35b1-4739-b662-b99c144ac029)

Thanks in anticipation for guidance .

**Comment by romainsacchi** [comment]
Hi, is it possible to have the full error? (view as scrollable element to see it full).

**Comment by zeeshankhan2** [comment]
Hi @romainsacchi  thanks for your response. But it was some back end libraries problem within my anaconda which is now working .thanks

**Comment by romainsacchi** [comment]
Can you elaborate, because some other users have this issue. Was it `pandas` or `xarray`?

**Comment by zeeshankhan2** [comment]
I just installed new anaconda and all setup ie premise.But now i am getting this error unfortunately.it works on one system but on another it gives error.
![image](https://github.com/polca/premise/assets/86916007/97f19ff6-d106-44aa-9678-70f299a36ac7)

**Comment by romainsacchi** [comment]
Hi @zeeshankhan2 
updating `pandas` and `xarray` (to `2023.9.0`), as the error indicates, should solve the issue.
Let me know.

**Comment by zeeshankhan2** [comment]
Updated xarray as below 
![image](https://github.com/polca/premise/assets/86916007/b6d990c0-8ff8-4f97-b554-0d8a24e09537)
 but still gives the same error.

**Comment by romainsacchi** [comment]
You can try to update premise to `1.7.2`.

**Comment by zeeshankhan2** [comment]
Can you please tell me how to update it and how to check its current version?

**Comment by romainsacchi** [comment]
You can do:

````
pip install premise==1.7.2
````

and to check the libraries installed you cna do:

````
pip list
````

**Comment by zeeshankhan2** [comment]
Got this error while installing 1.7.2
![image](https://github.com/polca/premise/assets/86916007/0f52bc82-1ec5-4c28-b6b3-101b26722556)
and it is same version after this

**Comment by romainsacchi** [comment]
you can also try conda

````
conda install -c conda-forge premise
````

**Comment by zeeshankhan2** [comment]
Is it normal thats it takes too much time using conda install -c conda-forge premise?
![image](https://github.com/polca/premise/assets/86916007/60f0d6da-bae6-4a91-94c0-105f1990f3e2)

**Comment by romainsacchi** [comment]
Yes, sometimes it takes a while. note that I just released `1.7.3` which fixes a bug.
The alternative, if you do not need to generate yourself the scenarios, is to use the [ScenarioLink](https://github.com/polca/ScenarioLink) plugin fro Activity Browser, to import the premise databases.

**Comment by zeeshankhan2** [comment]
Thanks for informing about the nice plugin.
I updated the premise to new version and now when i create databases and use ndb.update_electricity() and after thay when i am trying to write it to superstructure database using ndb.write_superstructure_db_to_brightway(),
 i am getting the following error although it didnot happened previously
![image](https://github.com/polca/premise/assets/86916007/5c657931-fc03-4232-a03f-45fed7428bc3)

**Comment by romainsacchi** [comment]
Ah yes, we fixed this issue in `1.7.6` -- sorry about that.

**Comment by romainsacchi** [comment]
Closing now. Re-open if the issue persists.
